recipes/repository.rb

```diff
@@ -19,68 +19,30 @@
 #
 
 case node['platform']
-when 'ubuntu'
-  if node['platform_version'].to_f <= 10.04
-    # Configure Brian's PPA
-    # We'll install php5-fpm from the Brian's PPA backports
-    apt_repository "brianmercer-php" do
-      uri "http://ppa.launchpad.net/brianmercer/php/ubuntu"
-      distribution node['lsb']['codename']
-      components ["main"]
-      keyserver "keyserver.ubuntu.com"
-      key "8D0DC64F"
-      action :add
-    end
-    # FIXME: apt-get update didn't trigger in above
-    execute "apt-get update"
-  end
 when 'debian'
   # Configure Dotdeb repos
   # TODO: move this to it's own 'dotdeb' cookbook?
   # http://www.dotdeb.org/instructions/
   if node['platform_version'].to_f >= 8.0
-    apt_repository "dotdeb" do
+    apt_repository 'dotdeb' do
       uri node['php-fpm']['dotdeb_repository']['uri']
-      distribution "jessie"
+      distribution 'jessie'
```

Comment:
There isn't support for Debian 9 right now so that would break it. We need to use the 8 packages